ansible-scylla-node/tasks/upgrade/main.yml

```diff
@@ -0,0 +1,305 @@
+---
+
+# ---[ TODO ]-----------------------------------------------
+#    0) Test better Ubuntu/Debian distros for specific version upgrades. Check if apt module should be used or not.
+#    1) Downgrade support
+#    2) Cross-edition upgrade/downgrade
+# ---------------------------------------------------------
+
+# Scylla matrix for supported distros, editions and versions
```

Comment:
this matrix likely needs a better place, since it can be shared with install too (e.g. refuse to install on old release)

Reply:
I agree, but so far 'upgrade' will be the only one to use it .